src/modules/AutoinstGeneral.rb

```diff
@@ -180,16 +186,14 @@ def Import(settings)
       settings = deep_copy(settings)
       SetModified()
       Builtins.y2milestone("General import: %1", settings)
-      @mode = Ops.get_map(settings, "mode", {})
-      @cio_ignore = Ops.get_boolean(settings, "cio_ignore", true)
-      @signature_handling = Ops.get_map(settings, "signature-handling", {})
-      @askList = Ops.get_list(settings, "ask-list", [])
-      @proposals = Ops.get_list(settings, "proposals", [])
-      @storage = Ops.get_map(settings, "storage", {})
+      @mode = settings.fetch("mode", {})
```

Comment:
Not sure if I get what you mean. Yes, it's a hash lookup... and that's exactly why `Hash#fetch` is used.